dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -12240,8 +12293,22 @@ bool DynamicsFitter::timeSyncAndInitializePipeline(
       previousTotalResidual = output.second;
       if (!output.first)
       {
-        residualMinimizationSuccess = false;
-        break;
+        std::cout << "zeroLinearResidualsAndOptimizeAngular() failed. "
+                     "useReactionWheels="
+                  << useReactionWheels << std::endl;
+        if (!useReactionWheels)
+        {
+          useReactionWheels = true;
+          detectUnmeasuredTorque = false;
+          i = -1;
+          iterations = 1;
+          continue;
```

Comment:
Should we remove the section in `engine.py` that re-runs this code if `timeSyncAndInitialize` fails now? It was essentially serving the same purpose as this.